test/init-ssr.spec.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @jest-environment node
+ */
+
+import axios from "axios";
+import {
+  createAccessToken,
+  createIdToken,
+  resetStore,
+} from "./config/utils.js";
+import { noMfaHeaders } from "./config/assertions.js";
+import Userfront from "../src/index.js";
+import { setMode } from "../src/mode.js";
+import { store } from "../src/store.js";
+import { apiUrl } from "../src/constants.js";
+
+const tenantId = "abcd5432";
+const domain = "com.example.myapp";
+
+jest.mock("axios");
+
+describe("init() method in Node/SSR environment (should not crash)", () => {
```

Comment:
How is the Node/SSR environment defined? Is it from the comment at the top?

Reply:
Yes - [Jest docs on it](https://jestjs.io/docs/configuration#testenvironment-string) for reference. The default environment is set in `jest.config.js` (for us it's `jsdom`) and the `@jest-environment` directive can override it per file.

(I think they did it this way for backwards compatibility and architectural reasons?)

I added a comment just below it to document the test file's purpose and link to the relevant Jest docs.